app.js

```diff
@@ -0,0 +1,115 @@
+// Section 2: JavasScript loops and expressions
+
+//Part 1 terms
+
+// Concatenation taking different strings and
+// adding/appending them to each other
+
+var age = 3;
+
+console.log("I'm " + age + " years old!"); 
+
+// interpolation formatting strings
+// you have a string and a substring that's meant to be replaced with a value
+// editing strings in place, as opposed to adding them to each other
```

Comment:
the key is that you're using variables in interpolation, whether you "add" strings with `+` or use the `${}` syntax